packages/javascript/src/environment.ts

```diff
@@ -15,6 +15,12 @@ export class Environment {
    * @return  {string}       The origin URL
    */
   public static origin(): string {
+    // We check for React Native here. React Native and Expo do
+    // not have an origin per se, so we handle that case.
+    if (navigator.product === "ReactNative" && !window.location) {
+      return "No origin available in React Native"
```

Comment:
Can we also return an empty string and log a warning or something else? Otherwise this string may end up somewhere it shouldn't be?

Reply:
Fixed!